tools/ci/push_explorer

```diff
@@ -25,7 +25,9 @@ tag_and_push() {
   # ie: docker tag and push smartcontract/explorer:0.9.1, since DOCKER_TAG=0.9.1
   export DOCKER_TAG=$1
   (set -x; docker tag smartcontract/explorer:circleci smartcontract/explorer:${DOCKER_TAG})
-  cd explorer && make dockerpush
+  cd explorer
```

Comment:
Should we use push/pop here instead since we just want to return to the original directory?

Reply:
I'm indifferent. I'll add the change.